tools/blocksconvert/builder/tsdb.go

```diff
@@ -223,16 +243,39 @@ func (d *tsdbBuilder) finishBlock(source string, labels map[string]string) (ulid
 
 	level.Info(d.log).Log("msg", "added symbols to index", "count", symbols)
 
-	stats, err := addSeriesToIndex(indexWriter, d.series)
+	unsortedChunksReader, err := chunks.NewDirReader(filepath.Join(d.tmpBlockDir, unsortedChunksDir), nil)
+	if err != nil {
+		return ulid.ULID{}, errors.Wrap(err, "chunks reader")
+	}
+	toClose[unsortedChunksReaderName] = unsortedChunksReader
+
+	chunksWriter, err := chunks.NewWriter(filepath.Join(d.tmpBlockDir, "chunks"))
+	if err != nil {
+		return ulid.ULID{}, errors.Wrap(err, "chunks reader")
```

Comment:
Here "chunks reader" is not correct.

```suggestion
		return ulid.ULID{}, errors.Wrap(err, chunksWriterName)
```